train.py

```diff
@@ -127,8 +127,8 @@ def get_bert(BERT_PT_PATH, bert_type, do_lower_case, no_pretraining):
 
     return model_bert, tokenizer, bert_config
 
-def get_opt(model, model_bert, fine_tune):
-    if fine_tune:
+def get_opt(model, model_bert, args):
```

Comment:
If the line indicated above in `predict.py` can be omitted, these change are not necessary I guess.